forest-node/ansible/ansible.cfg

```diff
@@ -0,0 +1,8 @@
+[defaults]
+host_key_checking = false
+inventory = hosts
+
+remote_user = chainsafe
+private_key_file = ~/id_rsa
+
+allow_world_readable_tmpfiles=true
```

Comment:
why?

Reply:
This is more like custom best practice, especially when running a new user (in our case, one with limited service privileges). So this command ensures that the temporary files created on the server are world readable and will display a level of warning instead of failing the running task.

It's more about general good practice with new users